src/engines/Cesium/core/Imagery.tsx

```diff
@@ -100,7 +100,7 @@ export function useImageryProviders({
 }): { providers: Providers; updated: boolean } {
   const newTile = useCallback(
     (t: Tile, ciat?: string) =>
-      presets[t.type || "default"]({
+      presets[isValidPresetTileType(t.type) ? t.type : "default"]({
```

Comment:
Key issue here, we didn't consider the case that t.type exists but not supported by presets.